src/scripts/main.js

```diff
@@ -1 +1,77 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', (e) => {
+    if (e.button === 0) {
+      resolve();
+    }
+  });
+
+  setTimeout(() => {
+    // eslint-disable-next-line prefer-promise-reject-errors
+    reject();
+  }, 3000);
+});
+
+const secondPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', (e) => {
+    if (e.button === 0 || e.button === 2) {
+      resolve();
+    }
+  });
```

Comment:
Consider removing the event listener after the promise is resolved to prevent memory leaks or unexpected behavior.